hashicorp/dev-portal
# feat: plug in data

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksplug-in-data-hashicorp.vercel.app/waypoint/tutorials/get-started-nomad-cloud/get-started-nomad-cloud) 🔎
- [Asana task](https://app.asana.com/0/1204333057896641/1204621999504630) 🎟️

## 🗒️ What

Plugs in the UI to the staging API with temp data. Adds the `VariantProvider` to the tutorial view. Refactors `VariantList` to use the `useVariant` hook.

<img width="1265" alt="Screenshot 2023-05-22 at 11 30 10 AM" src="https://github.com/hashicorp/dev-portal/assets/36613477/41dcf210-6932-4689-8834-ab92580d9ec1">

## 🧪 Testing

- Navigate [to this URL](https://dev-portal-git-ksplug-in-data-hashicorp.vercel.app/waypoint/tutorials/get-started-nomad-cloud/get-started-nomad-cloud)
- Verify that the variant UI renders
- Click 'Windows', verify that the URL and variant content updates.

## 💭 Anything else?
